<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entry</class>
 <widget class="QWidget" name="Entry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="website_layout">
     <item>
      <widget class="QLabel" name="website_label">
       <property name="text">
        <string>Website
Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="website_lineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="website_help">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="username_layout">
     <item>
      <widget class="QLabel" name="username_label">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username_lineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="username_help">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="password_layout">
     <item>
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="password_layout_2">
       <item>
        <widget class="QLineEdit" name="password_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="password_error_label">
         <property name="text">
          <string>Password cannot be empty.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="password_view_button">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="password_generator_button">
       <property name="text">
        <string>Generator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="password_help">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="repeat_layout">
     <item>
      <widget class="QLabel" name="repeat_label">
       <property name="text">
        <string>Repeat
Password</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="repeat_layout_2">
       <item>
        <widget class="QLineEdit" name="repeat_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="repeat_error_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Password is not the same.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="strength_layout">
     <item>
      <widget class="QLabel" name="strength_label">
       <property name="text">
        <string>Password
Strength</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="strength_progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="strength_help">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="url_layout">
     <item>
      <widget class="QLabel" name="url_label">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="url_lineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="url_help">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="twofa_layout">
     <item>
      <widget class="QLabel" name="twofa_label">
       <property name="text">
        <string>2FA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="twofa_lineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="twofa_help">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <item>
      <spacer name="button_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submit_button">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
